From db29f70205479f26b557d68db1a74fd0c7342b95 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd@kuhls.net>
Date: Tue, 3 Feb 2026 18:36:42 +0100
Subject: [PATCH] CMakeLists.txt: update CMake minimum required version

Building with CMake >= 4.x requires the cmake_minimum_required()
version to be at least 3.5, so let's update to that to fix:

CMake Error at CMakeLists.txt:3 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.

Upstream: https://gitlab.eclipse.org/eclipse/tcf/tcf.agent/-/merge_requests/13

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 agent/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/agent/CMakeLists.txt b/agent/CMakeLists.txt
index d5fa5396..aebf2cb5 100644
--- a/agent/CMakeLists.txt
+++ b/agent/CMakeLists.txt
@@ -1,6 +1,6 @@
 # -*- cmake -*-
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
 project(tcfagent C)
 include(GNUInstallDirs)
 
-- 
2.47.3

